lib/cylc/prerequisite.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, point, start_point=None):
         # cylc.cycling.PointBase
         self.start_point = start_point
 
-        # List of cycle point strings that this prerequiste depends on.
```

Comment:
That's definitely one of mine.  I think "prerequisite" but my hands/brain persist in typing "prerequiste".